src/TableColumn/Actions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace atk4\ui\TableColumn;
+
+/**
+ * Formatting money.
+ */
+class Actions extends Generic
+{
+    public $actions = [];
+
+    public function addAction($icon, $callback)
+    {
+        $cb = $this->add(new jsCallback(), $action);
```

Comment:
There is no variable $action and not used parameter $icon. I guess parameter should be named - $action.